### DISCUSSION TITLE
Add DataGrid support

### DISCUSSION THREAD

**Discussion by taozuhong** [CommunityToolkit/Maui#29] [Q&A] [answered]
Will DataGrid control will be support in this library?

**Comment by TheCodeTraveler** [comment, marked as answer]
Hi Taozuhong! Right now our focus is on porting the features in [Xamarin Community Toolkit](https://github.com/xamarin/xamarincommunitytoolkit) to this, the .NET MAUI Community Toolkit. This first phase will be complete in November when Microsoft releases .NET 6.

Xamarin Community Toolkit doesn't include a DataGrid control, which means that the first release candidate of the .NET MAUI Toolkit also will not include a DataGrid control.

That being said, we'd love for you to submit a design proposal! The Community Toolkit is not only created for, but also created by, the .NET Community.

We will begin reviewing/accepting design proposals in November following the release of .NET 6.

Stay tuned to the [.NET Blog](https://devblogs.microsoft.com/dotnet) where we will publish a blog post on how to contribute to the .NET Community Toolkit 👍

**Comment by inforithmics** [comment]
I think a starting point for a design proposal could be https://github.com/akgulebubekir/Xamarin.Forms.DataGrid

**Comment by AathifMahir** [comment]
Would love to see WindowsCommunityToolkit  DataGrid Control on Maui, As far as I looked through Dependencies on Windows Community Toolkit DataGrid Control, We have most of the replacement dependency on Maui. If you'll interested I'd be happy to help with a migration of Windows Community Toolkit DataGrid Control to Maui

https://github.com/CommunityToolkit/WindowsCommunityToolkit/tree/main/Microsoft.Toolkit.Uwp.UI.Controls.DataGrid

**Comment by CliffAgius** [comment]
Closed as answered

**Comment by vnraju** [comment]
Does .NET MAUI have a DataGrid in July 2024?

**Reply by bijington** [reply]
.NET MAUI does not neither does the .NET MAUI Community Toolkit (this repository)
